ts/src/management/tunnelManagementHttpClient.ts

```diff
@@ -672,9 +672,17 @@ export class TunnelManagementHttpClient implements TunnelManagementClient {
         this.raiseReportProgress(TunnelProgress.StartingRequestConfig);
         const config = await this.getAxiosRequestConfig(tunnel, options, accessTokenScopes);
         this.raiseReportProgress(TunnelProgress.StartingSendTunnelRequest);
-        const result = await this.request<TResult>(method, uri, body, config, allowNotFound);
-        this.raiseReportProgress(TunnelProgress.CompletedSendTunnelRequest);
-        return result;
+        try {
+            const result = await this.request<TResult>(method, uri, body, config, allowNotFound);
+            this.raiseReportProgress(TunnelProgress.CompletedSendTunnelRequest);
+            return result;
+        } catch (error) {
+            if (/certificate/i.test((error as Error).message)) {
+                throw new Error("Tunnel service HTTPS certificate is invalid. This may be caused by the use of a "+
```

Comment:
The original error message should be included also, since it might contain useful information.

Reply:
Included orignal message at the end of the new error message